ui/component/privacyAgreement/view.jsx

```diff
@@ -48,7 +48,7 @@ function PrivacyAgreement(props: Props) {
           <h1 className="section__title--large">{__('Welcome')}</h1>
           <h3 className="section__subtitle">
             {__(
-              `LBRY takes privacy and choice seriously. Just a few questions before you enter the land of content freedom. `
```

Comment:
Please remove the newline changes below. This is a good fix though.

Reply:
I don't understand what changes to remove?